wails-frontend/src/__ tests __/massaFormat.test.tsx

```diff
@@ -0,0 +1,93 @@
+import { toMAS } from '@massalabs/massa-web3';
+import { formatValue } from 'react-currency-input-field';
+
+export enum Unit {
+  MAS = 'MAS',
+  NanoMAS = 'NanoMAS',
+}
+
+//TODO : Implement better testing once we have a clear strategy
+
+// removeTrailingZerost and not rely on backtracking
+export function removeTrailingZeros(numStr: string): string {
+  return numStr.replace(/\.?0+$/, '');
+}
+
+export function formatStandard(
+  value: string | bigint,
+  unit = Unit.MAS,
+): string {
+  if (typeof value === 'bigint') {
+    value = value.toString();
+  }
+  const numInMas = unit === Unit.MAS ? value : toMAS(value).toString();
+
+  // Format the number as a string with separators
+  const formattedNum = formatValue({
+    value: numInMas,
+    groupSeparator: ',',
+    decimalSeparator: '.',
+    decimalScale: 9,
+  });
+  return removeTrailingZeros(formattedNum);
+}
```

Comment:
why redefining this?  you are not testing the actual code :/

Reply:
massFormat.tsx seems to host some code that is used else where, when i run the test cmd, it fails beacause jest can't find in correct module (which is generated by go in the ./wailsjs dir)

Reply:
<img width="579" alt="Screenshot 2023-09-18 at 14 20 52" src="https://github.com/massalabs/station-massa-wallet/assets/90157528/0f044065-e7bb-4055-94e2-0b95ae9e905f">